docs/gallery_code/oceanography/plot_load_nemo.py

```diff
@@ -13,24 +13,21 @@
 import iris
 import iris.plot as iplt
 import iris.quickplot as qplt
-from iris.util import promote_aux_coord_to_dim_coord
+from iris.util import equalise_attributes, promote_aux_coord_to_dim_coord
 
 
 def main():
     # Load the three files of sample NEMO data.
     fname = iris.sample_data_path("NEMO/nemo_1m_*.nc")
     cubes = iris.load(fname)
 
-    # Some attributes are unique to each file and must be blanked
-    # to allow concatenation.
-    differing_attrs = ["file_name", "name", "timeStamp", "TimeStamp"]
-    for cube in cubes:
-        for attribute in differing_attrs:
-            cube.attributes[attribute] = ""
+    # Some attributes are unique to each file and must be removed to allow
+    # concatenation.
+    equalise_attributes(cubes)
 
     # The cubes still cannot be concatenated because their time dimension is
-    # time_counter rather than time. time needs to be promoted to allow
-    # concatenation.
+    # time_counter, which has the same value for each cube, rather than time,
+    # which varies. time needs to be promoted to allow concatenation.
```

Comment:
I agree this was confusing. But I'm still finding it confusing!

Could you rephrase the whole thing to refer explicitly to `DimCoord`s and `AuxCoord`s? And it also hadn't occurred to me that promotion also demotes `time_counter` (perhaps I should have worked that one out); do you think it's worth mentioning that?

Reply:
Words are hard!  How's this? https://github.com/SciTools/iris/pull/4741/commits/848e53f1fcddae27f48f89fa78bfcc837dadda66

Note that, although the "time_counter" coord becomes an aux_coord on the cube, it is still of type `DimCoord` 🤪